webpack/assets/javascripts/react_app/pages/AuditsPage/AuditsPage.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Button, Icon } from 'patternfly-react';
+import { translate } from '../../common/I18n';
```

Comment:
Please use alias to `-` 
```js
import { translate as __ } from '../../common/I18n';
```
our string extractor uses `__` with regex

Reply:
bump?